### DISCUSSION TITLE
0.3.9 Release Candidate

### DISCUSSION THREAD

**Discussion by thelastWallE** [thelastWallE/OctoprintKlipperPlugin#63] [Releases]
### Discussions-thread for 0.3.9 RC

**0.3.9rc1:**
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc1

**0.3.9rc2:**
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc2

**0.3.9rc3:**
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc3

**0.3.9rc4:**
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc4

**0.3.9rc5:**
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc5

**0.3.9rc6:**
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc6

**Comment by thelastWallE** [comment]
### 0.3.9 RC2
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc2

**Comment by thelastWallE** [comment]
### 0.3.9 RC3
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc3

**Comment by Sineos** [comment]
Great updates. Appreciated.
Some feedback:
- printer.cfg is not set as default in settings nor in the editor
- after first install the editor window is misplaced towards the bottom of the screen. It is correct on further usage
- a "safe and close" button would be nice in the editor
- a checkbox to avoid the information about stopped prints would be nice

**Reply by thelastWallE** [reply]
- printer.cfg is not set as default in settings nor in the editor
  ->_I need to check this again it should have worked like this. On update to this version the configfile should be the baseconfig setting and the path of printer.cfg should be the configpath setting in the settings dialog of OctoKlipper.
Have you maybe the log where you updated to this rc?_

- a checkbox to avoid the information about stopped prints would be nice
  ->_a checkbox to hide those confirmation dialogs before a restart of klipper, if i understand correctly?_

**Reply by Sineos** [reply]
>     * printer.cfg is not set as default in settings nor in the editor
>       ->_I need to check this again it should have worked like this. On update to this version the configfile should be the baseconfig setting and the path of printer.cfg should be the configpath setting in the settings dialog of OctoKlipper.
>       Have you maybe the log where you updated to this rc?_

No, unfortunately, the log is already gone. I completely removed the plugin and reinstalled it and it behaved like it should. `printer.cfg` was there and the positioning of the window was correct. Maybe just an oddity on my side

>     * a checkbox to avoid the information about stopped prints would be nice
>       ->_a checkbox to hide those confirmation dialogs before a restart of klipper, if i understand correctly?_

Exactly

**Reply by thelastWallE** [reply]
> > ```
> > * a checkbox to avoid the information about stopped prints would be nice
> >   ->_a checkbox to hide those confirmation dialogs before a restart of klipper, if i understand correctly?_
> > ```
> 
> Exactly

- [x] https://github.com/thelastWallE/OctoprintKlipperPlugin/commit/bf7ebbd26126440510f53028130dc24a6926eee1

> a "safe and close" button would be nice in the editor

- [x] https://github.com/thelastWallE/OctoprintKlipperPlugin/commit/794b016bc3febd2d8a799f2ab7c8c9dcb636e5a7

**Comment by Sineos** [comment]
![grafik](https://user-images.githubusercontent.com/4459312/137452859-9eb6bc0b-f091-4c4d-acd4-5ff365f8e50b.png)

May I propose moving down the Plugin Config to Tools. It is rarely used and (more importantly) I'm too stupid and always hit this button instead of opening `printer.cfg` editor

**Reply by thelastWallE** [reply]
- [x] create little wrench symbol next to menu

**Reply by thelastWallE** [reply]
https://github.com/thelastWallE/OctoprintKlipperPlugin/commit/b0ee099a41360667dce6469a24e4fa02c7a524ae

**Comment by Sineos** [comment]
Very nice. Thanks a lot. I noticed two small things whenn installing current RC git:
* Apparently the chosen font size in the editor (via +/- buttons) do not survive an OctoPrint restart
* The reload on save defaulted to off. Default should be on IMO
![grafik](https://user-images.githubusercontent.com/4459312/138074429-3bbd2a9e-e967-4b01-a222-bda6c0b9ff67.png)

**Reply by thelastWallE** [reply]
Just pushed to rc branch. You can try with a manual installation of the zip file in OctoPrint with the Plugin Manager.
https://github.com/thelastWallE/OctoprintKlipperPlugin/archive/c9b2d3a49b4a606ad92fb3ba581125fb8b0145de.zip

**Comment by Sineos** [comment]
Seems to work very well. Did not notice any additional items. Many thanks for your commitment. 👍

**Comment by thelastWallE** [comment]
### 0.3.9RC4
https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc4

**Comment by thelastWallE** [comment]
### 0.3.9RC5

https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc5

**Comment by thelastWallE** [comment]
### 0.3.9RC6

https://github.com/thelastWallE/OctoprintKlipperPlugin/releases/tag/0.3.9rc6
